src/Unleash/Unleash.csproj

```diff
@@ -72,7 +72,7 @@
     <PackageReference Include="murmurhash" Version="1.0.3" />
     <PackageReference Include="NuGet.Versioning" Version="6.1.0" />
     <PackageReference Include="System.Text.Json" Version="8.0.5" />
-    <PackageReference Include="Unleash.Yggdrasil" Version="1.0.4" />
+    <PackageReference Include="Unleash.Yggdrasil" Version="1.0.5-beta.0" />
```

Comment:
beta.0 or beta.1?

Reply:
beta.0 was correct, yeah. beta.1 was a build to get past the nuget verify issue

That being said, I'm taking this up to stable 1.0.5